app/templates/-current-user-header.hbs

```diff
@@ -6,34 +6,18 @@
 
     <nav>
       <ul class="nav nav-pills navbar-right user-nav">
-        <li>
-          {{#if model.organization}}
-            {{link-to model.organization.name "index"}}
-          {{else}}
-            {{link-to model.stack.organization.name "index"}}
-          {{/if}}
-        </li>
-
         <li class="dropdown current-user">
           {{#dashboard-dropdown title=session.currentUser.name defaultPath="index"}}
 
-            {{#dashboard-dropdown-item}}
-              {{link-to "Dashboard" "index"}}
-            {{/dashboard-dropdown-item}}
-
-            {{#dashboard-dropdown-item}}
-              {{#link-to "settings"}}Settings{{/link-to}}
-            {{/dashboard-dropdown-item}}
+            {{#each organization in session.currentUser.organizations}}
```

Comment:
The block params style is preferred:

```handlebars
{{#each session.currentUser.organizations as |organization|}}
  {{dashboard-dropdown-organization-menu organization=organization}}
  {{dashboard-dropdown-divider}}
{{/each}}
```

Though not required. The `in` is basically remapped at compile time to what I wrote out here.